index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Code The Future</title>
+</head>
+<body>
+    <div>
+        <div>
+            <h1>Code The Future</h1>
+            <nav>
+                <ul>
+                   <li><a href="">Lorem</a></li>
```

Comment:
Great work done @cherishebojudah 
For the link, I think you can put in the texts now since we already have a list of career paths.

Reply:
Thank you. That would be done by my next commit.